vital/types/uid.cxx

```diff
@@ -68,11 +68,9 @@ is_valid() const
 
 
 // ------------------------------------------------------------------
-const char*
-uid::
-value() const
+std::string const& kwiver::vital::uid::value() const
 {
```

Comment:
Oh... I guess my IDE *did* do something when I told it to "update definition" after changing the declaration in `uid.h`... not the *right* thing, but...

Fixed.